### PR TITLE
Update the plugin version to next minor version

### PR DESCRIPTION
**Pull request by ThisaruGuruge**
## Purpose
$Subject

This is to provide functionality to build Ballerina packages from the Docker images using the Ballerina Gradle Plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,57 @@
 
 ## Introduction
 
-The Ballerina Gradle plugin is designed to work
-similar to the Java Gradle plugin. This adds ballerina project bundling functionalities along with testing.
+The Ballerina Gradle plugin is used to build Ballerina modules using Gradle. This plugin is used in the Ballerina platform to build the Ballerina Libraries. This plugin is needed when there is a Gradle project with a Ballerina submodule. The Ballerina submodule can be built using this plugin.
+
+>**Note:** This plugin is not used to build generic Ballerina projects. To build generic Ballerina projects, use the Ballerina CLI.
```
Comment (NipunaRanasinghe) [inline README.md:7]:
```suggestion
>**Note:** This plugin is not recommended for building standalone Ballerina packages. It is recommended to use the Ballerina CLI directly in such scenarios.
```

**Review by NipunaRanasinghe — APPROVED** [review]

File: src/main/groovy/io/ballerina/plugin/BallerinaPlugin.groovy [hunk 2]
```diff
@@ -90,15 +93,15 @@ class BallerinaPlugin implements Plugin<Project> {
         }
 
         project.dependencies {
-            if (checkForBreakingChanges) {
+            if (buildOnDocker || ballerinaExtension.buildOnDockerImage) {
```
[the remark this answers is not in the document]
Comment (ThisaruGuruge) [reply src/main/groovy/io/ballerina/plugin/BallerinaPlugin.groovy:96]: Yeah, I tried that way first, but it doesn't work. As per the Gradle docs, the plugin extension scope is limited to registered tasks. We cannot access them outside. That's the reason for duplicating the logic. Couldn't find a better way to handle it.

[the remark this answers is not in the document]
Comment (ThisaruGuruge) [reply src/main/groovy/io/ballerina/plugin/BallerinaPlugin.groovy]: Found another way. Updated this.

**Review by NipunaRanasinghe — APPROVED** [review]
LGTM

**Pull request merged**